crates/sats/src/convert.rs

```diff
@@ -48,3 +49,27 @@ built_in_into!(f32, F32);
 built_in_into!(f64, F64);
 built_in_into!(&str, String);
 built_in_into!(&[u8], Bytes);
+
```

Comment:
Here and some places it looks like `ConstraintId` isn't handled? I'm assuming `ConstraitId` is a type that we expect to use a lot more of shortly? And so it's named but not always plumbed all the way through etc

If that take is correct, I'm indifferent between you adding the `impl` for it here now, or later when it's need.

Reply:
Good catch. I have that in my PRs but here this move to another crate so I miss it.